Misc/NEWS.d/next/Library/2022-01-11-15-54-15.bpo-46333.B1faiF.rst

```diff
@@ -0,0 +1 @@
+Honor the ``module`` parameter in :class:`typing.ForwardRef`, as it carries semantic information about the forward reference. This affects the :meth:`__eq__`, :meth:`__repr__` and :meth:`__hash__` methods of the class.
```

Comment:
Docs should be < 80 characters please. I also suggest the following rearrangement to make things more obvious to a user reading this.
```suggestion
The :meth:`__eq__`, :meth:`__repr__` and :meth:`__hash__` methods of
:class:`typing.ForwardRef` now honor the ``module`` parameter of
:class:`typing.ForwardRef`. Forward references from different
modules are now differentiated.
```